projects/plugins/jetpack/modules/masterbar/admin-menu/load.php

```diff
@@ -2,22 +2,30 @@
 /**
  * Admin Menu loader.
  *
+ * @deprecated $$next-version$$
+ *
  * @package Jetpack
  */
```

Comment:
You might throw a
```suggestion
 * @phan-file-suppress PhanDeprecatedFunction -- Ok for deprecated code to call other deprecated code.
 */
```
in here to avoid having a bunch of additions to the baseline. Same for other relevant files, particularly the tests.

Reply:
Done via https://github.com/Automattic/jetpack/pull/38109/commits/526527e02d687d15c82e410074b11b78710f8752